src/extension/ncwms/src/test/java/au/org/emii/geoserver/wms/NcwmsTest.java

```diff
@@ -2,57 +2,19 @@
 
 import org.dom4j.Document;
 import org.dom4j.io.SAXReader;
-import org.junit.Test;
 
 import java.io.FileInputStream;
-import java.util.*;
 
-import static org.apache.wicket.util.tester.WicketTesterHelper.assertEquals;
 
 public class NcwmsTest {
     private Document getCapabilitiesDocument() throws Exception {
         SAXReader reader = new SAXReader();
         return reader.read(new FileInputStream("./src/test/resources/get_capabilities.xml"));
     }
 
-    @Test
-    public void testGetSupportedStyles() throws Exception {
-        List<String> expectedStyles = new ArrayList<String>() {{
-            add("barb");
-            add("fancyvec");
-            add("trivec");
-            add("stumpvec");
-            add("linevec");
-            add("vector");
-            add("boxfill");
-        }};
-        Collections.sort(expectedStyles);
-
-        List<String> returnedStyles = Ncwms.getSupportedStyles(getCapabilitiesDocument(), "sea_water_velocity");
-        Collections.sort(returnedStyles);
-
-        assertEquals(expectedStyles, returnedStyles);
+    private Document getMetadataDocument() throws Exception {
+        SAXReader reader = new SAXReader();
+        return reader.read(new FileInputStream("./src/test/resources/get_metadata.json"));
```

Comment:
I realize it's just a test, but should we be closing the file, after reading it?